### DISCUSSION TITLE
Incrementing a value?

### DISCUSSION THREAD

**Discussion by pndalal** [GodotNuts/GodotFirebase#226] [Q&A] [answered]
My game needs to increment the value of an existing field safely.

I see from the Firestore [documentation](https://cloud.google.com/firestore/docs/samples/firestore-data-set-numeric-increment) that you can increment a field by writing `firestore.Increment({value})`

Does the plugin currently support this? I couldn't find anything in the documentation, but I wanted to check here just in case.

**Comment by BearDooks** [comment, marked as answer]
@pndalal  Currently we do not support this since I don't know if the REST API does. The best way would be to store the value in a variable in Godot, increment it, and then store it back in firebase.

Thanks

Chuck

**Reply by pndalal** [reply]
Thanks for the quick reply! Eventually I'll use a cloud function, so this shouldn't be an issue.

**Reply by BearDooks** [reply]
Awesome, that makes sense.
